File: 8ku/TotalAmountOfPoints/TotalAmountOfPoints.java
public class TotalPoints {
  
    public static int points(String[] games) {
        Integer result = 0;
      for (String str : games) {
        String arr = str.split(":")[0];
        if (Integer.parseInt(str.split(":")[0]) > Integer.parseInt(str.split(":")[1])) {
            result += 3;
        }else if (Integer.parseInt(str.split(":")[0]) == Integer.parseInt(str.split(":")[1])) {
            result += 1;
        }
      }
      return result;
    }
}

File: 8ku/TotalAmountOfPoints/TotalAmountOfPoints.go
package kata

import "strings"

func Points(games []string) int {
	result := 0
	for _, v := range games {
		if strings.Split(v, ":")[0] > strings.Split(v, ":")[1] {
			result += 3
		} else if strings.Split(v, ":")[0] == strings.Split(v, ":")[1] {
			result += 1
		}
	}
	return result
}
